.github/workflows/ci.yml

```diff
@@ -75,12 +60,6 @@ jobs:
         path: target
         key: ${{ matrix.os }}-${{ matrix.rust }}-cargo-build-target-${{ hashFiles('**/Cargo.toml') }}
 
-    - name: check
-      uses: actions-rs/cargo@v1
-      with:
-        command:  check
-        args: --workspace --all --bins --examples --tests
```

Comment:
please don't remove this

Reply:
it was just a test but it failed.  gh actions sucks.